fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.py

```diff
@@ -54,17 +58,66 @@ class FOSSUserProfile(WebsiteGenerator):
         website: DF.Data | None
         x: DF.Data | None
         youtube: DF.Data | None
-
     # end: auto-generated types
+
     def validate(self):
         self.validate_username()
         self.set_route()
 
+    def on_update(self):
+        prev_user_doc = self.get_doc_before_save()
+        if prev_user_doc is None:
+            return
+        try:
+            if (
+                self.full_name
+                is not self.get_doc_before_save().full_name
+            ):
+                frappe.db.set_value(
+                    "User",
+                    {"email": self.email},
+                    "full_name",
+                    self.full_name,
+                )
+            if (
+                self.username
+                is not self.get_doc_before_save().username
+            ):
+                frappe.db.set_value(
+                    "User",
+                    {"email": self.email},
+                    "username",
+                    self.username,
+                )
```

Comment:
Let's try this with ignore permission too if possible

Reply:
The case is the same as demoed in the previous comment. It doesn't explicitly make a difference I think in our use-case.